data/tlg0007/tlg082b/tlg0007.tlg082b.perseus-grc3.xml

```diff
@@ -0,0 +1,373 @@
+<pubPlace>Medford, MA</pubPlace>
+<authority>Perseus Project</authority>
+<date type="release">2010-12-13</date></publicationStmt>
+<sourceDesc>
+<biblStruct>
+<monogr>
+<author>Plutarch</author>
+<title xml:lang="lat">Moralia</title>
+<respStmt>
+<resp>with an English Translation by</resp>
+<name>Frank Cole Babbitt</name>
+</respStmt>
+<imprint>
+<pubPlace>Cambridge, MA</pubPlace>
+<publisher>Harvard University Press</publisher>
+<pubPlace>London</pubPlace>
+<publisher>William Heinemann Ltd.</publisher>
+<date type="printing">1931</date>
+</imprint>
+<biblScope unit="volume">3</biblScope>
+</monogr>
+<ref target="https://archive.org/details/L245PlutarchMoraliaIIIKingsCommandersRomansSpartans/page/n469">The Internet Archive</ref></biblStruct>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+<editorialDecl>
+<correction method="silent">
+<p>optical character recognition</p>
+</correction>
+</editorialDecl>
+<p>Text encoded in accordance with the latest EpiDoc standards</p><p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE Architecture</p><refsDecl n="CTS">
+<!--<state delim="." unit="chapter" />
+<state unit="section" />-->
+<cRefPattern matchPattern="(\w+)" n="section" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"><p>This pointer pattern extracts sections</p></cRefPattern></refsDecl>
+<!--<refsDecl doctype="TEI.2">
+<state n="chunk" unit="stephpage" />
+</refsDecl>-->
+</encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="eng">English</language>
+<language ident="lat">Latin</language>
+</langUsage>
+</profileDesc>
+<revisionDesc>
+<change who="RS" when="2010-05">tagged and parsed</change>
+<change who="Sophia Elzie" when="2019-07-01">EpiDoc and CTS Compliance</change>
+</revisionDesc>
+</teiHeader>
+<text xml:lang="grc">
+<body>
+<pb xml:id="v.3.p.454" />
+<div n="urn:cts:greekLit:tlg0007.tlg082b.perseus-grc3" type="edition" xml:lang="grc">
+<head>Ἀργιλεωνίδος<note xml:lang="eng" anchored="true" place="unspecified" resp="ed">The title, <foreign xml:lang="grc">λακαινων αποφθεγματα</foreign>, and the headings, <foreign xml:lang="grc">αργιλεωνιδος</foreign>, etc., are almost always omitted in the mss.</note>
+</head>
+<div n="1" subtype="section" type="textpart">
+<p rend="indent">
+<milestone n="240c" unit="stephpage" xml:id="stephpage-240c"/>
+Ἀργιλεωνὶς ἡ Βρασίδου μήτηρ, τελευτήσαντος αὐτῇ τοῦ υἱοῦ, ὡς παραγενόμενοί τινες τῶν Ἀμφιπολιτῶν εἰς Σπάρτην ἧκον πρὸς αὐτὴν ἠρώτησεν
+εἰ καλῶς καὶ ἀξίως τῆς Σπάρτης ὁ υἱὸς ἐτελεύτα· μεγαλυνόντων δʼ
+ἐκεῖνον καὶ λεγόντων ἄριστον ἐν τοῖς τοιούτοις ἔργοις ἁπάντων Λακεδαιμονίων εἶναι, εἶπεν, <q type="unspecified">ὦ ξένοι, καλὸς μὲν ἦν κἀγαθὸς ὁ παῖς μου, πολλοὺς δʼ ἄνδρας Λακεδαίμων ἔχει
+<milestone n="240d" unit="stephpage" xml:id="stephpage-240d" /> τήνω κάρρονας.</q>
+</p>
+</div>
+</div>
+<div n="urn:cts:greekLit:tlg0007.tlg082b.perseus-grc1" type="edition" xml:lang="grc">
```

Comment:
it looks like the top level div  is repeated here? did you intend to do chapter/section instead?